style.css

```diff
@@ -11,6 +16,9 @@ h1 {
   font-size: 40px;
   
 
+}
+p{
+text-align: center;
```

Comment:
Make sure you're careful with indentation; it can be tricky to figure out what's happening later when indentation is weird early in working on a project.